src/redux/reducers/cellSets/helpers.js

```diff
@@ -28,14 +30,18 @@ const createPropertiesFromTree = (data) => {
           type,
         };
 
+        if (!rootNode && !_.isNil(parentNode)) {
```

Comment:
Why do you use `!_.isNil(parentNode)`? Can't we do the same check with `!(parentNode)`

Reply:
With `isNil` we are checking for a `null` or `undefined`, so it is more correct compared to using !`parentNode` which also allows `0` and `false`. 

I know we could assume `0` and `false` never happens, but with javascript it unfortunately it can happen and sometimes we end up with falses or something. I prefer to go with the more strict option when possible so that we catch those `false` or `0` that shouldn't be there.

Reply:
Are the values `0` and `false` possible in this case?

Reply:
In this case, parent node would be an object if it's defined right?

Reply:
if defined it would be a string